src/main/resources/db/migration/V8__add_event_stablecoin.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE event ADD COLUMN stable_coin_decimals NUMERIC NOT NULL DEFAULT 6;
```

Comment:
You can add precision to `NUMERIC` here since we know that the biggest number in contracts is of type `uint256`. `2^256 = 1.1579209e+77` so using `NUMERIC(78)` would be sufficient.

Reply:
For this value biggest number is 27. https://github.com/web3j/web3j/blob/d175e36f4800dfb885455f320048f98563052b45/utils/src/main/java/org/web3j/utils/Convert.java#L47
Maybe set `smallint`?